vimeo/vimeo-networking-java
# VA-4212 Publish to Social Jobs

#### Issue
- https://vimean.atlassian.net/browse/VA-4212
- https://vimean.atlassian.net/browse/VA-4213

#### Summary
Added support for GET and PUT requests to /videos/{video_id}/publish_to_social

#### How to Test
Test setup is a bit complicated. See me IRL if you want instructions for testing in the app.
